lib/biscuit.rb

```diff
@@ -1,9 +1,13 @@
 require "biscuit/version"
+require "biscuit/secrets_decrypter"
+
+require "open3"
+require "yaml"
 
 module Biscuit
   def self.run!(command)
-    result = `#{__dir__}/../bin/_biscuit #{command}`
-    raise(result.slice(0, 200)) unless $?.success?
-    result
+    stdout, stderr, status = Open3.capture3("#{__dir__}/../bin/_biscuit #{command}")
+    raise(stderr.slice(0, 200)) unless status == 0
```

Comment:
A difference (which may be an improvement) - previously we raised with a slice of the *stderr* output, but you have changed it to a slice of the *stdout* output. That seems likely to be more correct, but I wanted to make sure you were aware, and had checked out the actual behavior around this.

Reply:
`Open3.capture3` is a *way* better tool for this by the way, even without the stubbing benefits :-)

Reply:
Would you mind making this into a named exception? (maybe `Biscuit::Error`)

Reply:
@nevinera previously the result holds both stdout and stderr I believe. maybe it should be `Biscuit::Error.new("#{stdout} #{stderr}".slice(0, 200))` thoughts?

Reply:
never mind,  it is `stdout` only